client/src/component/item.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import Item from './item';
+
+describe('item test', () => {
+    const remove = jest.fn();
+    const update = jest.fn();
+    const propData = {
+        id: '',
+        itemName: '',
+        repair: ' ',
+    }
+    render(<Item item={propData} remove={remove} updateItemList={update}/>);
+    const delIcon = screen.getByTestId('delIcon')
+
+    test('renders Item', () => {
+        expect(screen.getByTestId('nameTxtField')).toBeInTheDocument;
+    });
+
+    test('onChange ', () => {
+        delIcon.onclick = remove;
+        fireEvent.click(delIcon);
+        expect(remove).toHaveBeenCalledTimes(1);
+    })
```

Comment:
It should be safe to assume this is tested since you're using their component.

Reply:
So is this test unnecessary?

Reply:
Yeah, I think so.